sdk/storage/Azure.Storage.Common/src/Sas/SasQueryParameters.cs

```diff
@@ -60,6 +61,18 @@ public partial class SasQueryParameters
         // sig
         private string _signature;
 
+        // saoid
+        private string _authorizedAadObjectId;
+
+        /// suoid
+        private string _unauthorizedAadObjectId;
+
+        /// scid
+        private string _correlationId;
+
+        // sdd
+        private uint? _directoryDepth;
```

Comment:
why `uint`?

Reply:
I think cause directory depth can only be 0 and above